src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php

```diff
@@ -380,7 +381,7 @@ public function testTableNotExists(): void
         $client = $this->getClient();
 
         $input = new DescribeTableInput([
-            'TableName' => $this->tableName,
+            'TableName' => $this->tableName.'_non_exist',
```

Comment:
This is wrong, the test asserts that `tableNotExists($input)` is false. which mean the table should exists

Reply:
You are correct. I didnt see that until now. 

Thank you